plugins/css-prefers-color-scheme/package.json

```diff
@@ -25,15 +25,14 @@
   },
   "files": [
     "CHANGELOG.md",
-    "INSTALL.md",
```

Comment:
Is this intended?

Reply:
Yes, I don't think it is super useful to have an `INSTALL.md` somewhere buried in your node_modules. I might be wrong about this and will change it back if you prefer keeping it :)

Reply:
Great! Resolving